### PR TITLE
Bugfix copyright year and FB button

### PR DESCRIPTION
**Pull request by BlackMorzan**
Simple changes in Views to fix bugs #385 and #386 

## Description
Changed copyright year
Commented out FB event button

## Where should the reviewer start?
Doens't matter

## Motivation and context
Fixes for issue #385 and #386

## How has this been tested?
Looks fine (see in screenshots)

## Screenshots (if appropriate)
FB_Button

## Types of changes
Simple changes in cshtml files
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Home/Contact.cshtml [hunk 1]
```diff
@@ -38,11 +38,13 @@
                             <span class="icon-social-round icon-social-dark icofont icofont-social-youtube-play"></span>
                         </a>
                     </p>
-                    <p>
+                    @*
+                    <<p>
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Home/Contact.cshtml:42]: there is a duplicate < sign

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Shared/_Layout.cshtml [hunk 2]
```diff
@@ -147,7 +147,7 @@
                 <div class="col-md-4 col-sm-4 ">
                     <div class="legal_text">
                         <p>
-                            © 2019 - DevAdventCalendar<br />
+                            © 2020 - DevAdventCalendar<br />
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Shared/_Layout.cshtml:150]: please use @DateTime.Now.Year

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Home/Contact.cshtml (hunk 1 above)
Comment (BlackMorzan) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Home/Contact.cshtml:42]: Fixed

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Shared/_Layout.cshtml (hunk 2 above)
Comment (BlackMorzan) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Shared/_Layout.cshtml:150]: Thats better idea - already changed

**Review by WTobor — APPROVED** [review]

**Pull request merged**